suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -19,6 +19,8 @@ tests:
       abort-on-fail: true
       module: install_prereq.py
       name: install ceph pre-requisites
+      config:
```

Comment:
Please remove this test necessary config.

Reply:
Done